lang/go/go.talon

```diff
@@ -66,25 +113,46 @@ else <user.text> [over]:
     key("enter")
     insert(user.formatted_text(text, "PRIVATE_CAMEL_CASE"))
 
-state while: "while "
```

Comment:
why did you remove this one?  Fwiw this should probably become a keyword using the keyword mechanism we have for other languages

Reply:
There is no while keyword in go: https://golangdocs.com/while-loop-in-golang